pkgs/by-name/so/sonarr/package.nix

```
@@ -129,10 +150,11 @@ buildDotnetModule {
       # makes real HTTP requests
       "FullyQualifiedName!~NzbDrone.Core.Test.TvTests.RefreshEpisodeServiceFixture"
       "FullyQualifiedName!~NzbDrone.Core.Test.UpdateTests.UpdatePackageProviderFixture"
-
+    ] ++ lib.optionals stdenvNoCC.hostPlatform.isDarwin [
```

```suggestion
    ] ++ lib.optionals stdenvNoCC.buildPlatform.isDarwin [
```

IIRC these tests fail when run on macOS, not when building for macOS.

---

Good catch, fixed :)